nginx/nginx.conf

```diff
@@ -0,0 +1,16 @@
+upstream flask {
+    server rdf_differ:3040;
```

Comment:
this should probably not be hardcoded.

Reply:
possibly port forwarding shall be left for the next level of infrastructure manipulation, but maybe I'm wrong.